src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -21,6 +21,9 @@ object ScalafixPlugin extends AutoPlugin {
   object autoImport {
     val Scalafix = Tags.Tag("scalafix")
 
+    val ScalafixConfig = config("scalafix")
```

Comment:
This is the third type of "scalafix config" after the function returning sbt settings applied to `Compile`/`Test` & the sbt setting referencing the actual configuration file for scalafix... I initially considered naming it `Scalafix` and deleting/renaming the current one 2 lines above since it's not used internally since 4dbaf1ece9a67ab686cf5c4e12483298996005de, but that would be a breaking change.